### PR TITLE
Meal-sharing endpoints

### PR DESCRIPTION
**Pull request by katayouny**

### CODE REVIEW

File: src/backend/app.js [hunk 1]
```diff
@@ -21,10 +24,121 @@ app.use(cors());
...
+
+const todayDate = new Date();
+app.get("/future-meals", async (req, res) => {
+  try {
+    const futureMeals = await knex
+      .select("*")
+      .from("meal")
+      .where("meal_time", ">", todayDate);
+    if (futureMeals.lenght > 0) {
+      res.status(200).json(futureMeals);
```
Comment (LucasIversen) [inline src/backend/app.js:41]: good!

File: src/backend/app.js [hunk 2]
```diff
@@ -21,10 +24,121 @@ app.use(cors());
...
+});
+
+//first-meal	Respond with the first meal (meaning with the minimum id) -respond with a single meal, meaning an object
+app.get("/first-meal", async (req, res) => {
+  try {
+    const firstMeal = await knex
+      .select("*")
+      .from("meal")
+      .orderBy("id")
+      .limit(1); // Limiting the result to just one meal
```
Comment (LucasIversen) [inline src/backend/app.js:92]: Goood!!

**Review by LucasIversen — APPROVED** [review]
Really good job! No comments from me 👌

**Comment by katayouny** [comment]
Hi @LucasIversen 
Thank you so much.

**Pull request merged**